README.md

```diff
@@ -264,39 +264,18 @@ The API tests run after smoke tests.
 
 #### Clear Helm Release
 
-If your service never uses the deployed resources after the pipeline is successful or if you refer to them only for logs in case of failure, you can uninstall the helm release to free resources on the cluster. Note that clearing the helm release won't deny you access to the pods logs, as they are saved as artefacts in Jenkins before the helm release is cleared.
-
-
-To clear the helm release regardless of success or failure, do the following:
-
-  ```
-  withPipeline(type, product, component) {
-    ...
-    enableCleanupOfHelmReleaseAlways()
-    ...
-  }
-  ```
-
-To clear the helm release on a successful build, do the following:
-
+- By default your Helm resources are uninstalled to free up resources on the cluster.
+- You can keep these resources by adding the **enable_keep_helm** label on your PR.
```

Comment:
the below line doesn't look right, it says it's for master but won't it do it for master and PRs?

Reply:
See https://github.com/hmcts/cnp-jenkins-library/pull/1105/commits/4455041bfbe163baef2605c89fe6e574dc81f29f